__tests__/Calculator.test.js

```diff
@@ -0,0 +1,47 @@
+import Calculator from '../src/calculator';
+
+describe('계산기', () => {
+  it('2개의 숫자에 대해 덧셈이 가능하다', () => {
+    const calculator = new Calculator();
+
+    expect(calculator.sum(1, 2)).toBe(1 + 2);
+    expect(calculator.sum(-1, -1)).toBe(-1 + -1);
```

Comment:
```suggestion
    expect(calculator.sum(1, 2)).toBe(3);
    expect(calculator.sum(-1, -1)).toBe(-2);
```
이렇게 표현해줘야 테스트코드를 보는사람 입장에서 input에 대한 output을 이해하기가 더 쉬울 것 같아요!